scipy/stats/_rvs_sampling.py

```diff
@@ -117,9 +124,6 @@ def rvs_ratio_uniforms(pdf, umax, vmin, vmax, size=1, c=0, random_state=None):
     >>> stats.kstest(rvs, 'expon')[1]
     0.928454552559516
 
-    Sometimes it can be helpful to use a non-zero shift parameter `c`, see e.g.
-    [2]_ above in the case of the generalized inverse Gaussian distribution.
```

Comment:
I can't see from the changes in PR alone if the second reference is still used in the docstring. If it isn't used, should the ref itself also be deleted? (if we still cite in the docstring then safe to keep)

Reply:
it is used on l. 47

Reply:
ok, thanks for clarifying.